### ISSUE TITLE
Grim Dawn doesn't respect vsync

### ISSUE THREAD

**Issue by clapbr** [kakra/wine-proton#18]
Using /d3d11 (dxvk), Grim Dawn x64 doesn't enable vsync. Working fine with valve's proton and wine-staging. Any idea what could it be? I don't have other dxvk game to test if it's game specific.

**Comment by ahjolinna** [comment]
could you add some log and your system spec? (inxi -b)

to enable logs; just rename the `user_settings.sample.py` --> [user_settings.py](https://gist.githubusercontent.com/ahjolinna/977763a9a0f64fccf7907f5363764112/raw/f60f7d41d292c38f036037c0560046431788acfe/user_settings.py) (check what I have enable in it for reference)
and then add `PROTON_LOG=1 %command%` to the games launch option

**Comment by kakra** [comment]
Yes, there's a commit in my branch that may convert vsync into triple buffering. This would decouple the maximum frame rate rendered from that of the monitor while maintaining tear free displaying of frames.

What exactly do you observe?

BTW: it's in the dxvk subproject...

**Comment by clapbr** [comment]
Indeed it has no tearing, but it microstutters/drop frames constantly on this game at least. It also prevents me from using Freesync since it stays out of it's range. 

This game specifically has triple buffering also and it works fine. It would be useful to be able to switch this feature on/off at runtime, meanwhile I'll do a custom build without it.

**Comment by kakra** [comment]
It should be enough to just replace dxvk. But I'm thinking of removing that commit because it no longer provides the benefits it did for me some versions ago.

**Comment by clapbr** [comment]
Your call, feel free to close this when you want.

**Comment by kakra** [comment]
Let me test Grim Dawn myself first... I'm going to buy it soonish... (well, whatever "soon" could mean in my current life)
